src/app/pages/Login.tsx

```diff
@@ -23,35 +29,81 @@ function Login() {
     history.push("/signup") 
   }
 
+  const useStyles = makeStyles((theme) => ({
+    paper: {
+      marginTop: theme.spacing(15),
+      display: 'flex',
+      flexDirection: 'column',
+      alignItems: 'center',
+    },
+    form: {
+      width: '100%', // Fix IE 11 issue.
+      marginTop: theme.spacing(3),
+    },
+    submit: {
+      margin: theme.spacing(3, 0, 2),
+    },
+  }));
+
+    const classes = useStyles();
+
   return (
-    <div>
-      <h2>Login</h2>
-
-      <form onSubmit={handleLogin}>
-        <div>
-          <input
-            placeholder="username"
-            onChange={e => setUsername(e.target.value)}
-            value={username}
-            name="username"
-          />
-        </div>
+      <Container component="main" maxWidth="xs">
+        <div className={classes.paper}>
+          <img alt='stave logo' src='https://github.com/asyml/stave/blob/master/public/Stave-dark-text@1x.png?raw=true'/>
```

Comment:
The correct way to access the file to use the public URL like this:

https://create-react-app.dev/docs/using-the-public-folder/

The difference is, the project wouldn't rely on the service status of "Github". Instead, the logo file will be deployed to the user's server.

Reply:
Or you can follow the following:

https://create-react-app.dev/docs/adding-images-fonts-and-files/

But using the github link is not recommended.